src/plugins/intel_npu/src/compiler/src/zero_compiler_in_driver.cpp

```diff
@@ -619,9 +627,10 @@ std::unordered_set<std::string> LevelZeroCompilerInDriver<TableExtension>::getQu
 }
 
 template <typename TableExtension>
-std::vector<uint8_t> LevelZeroCompilerInDriver<TableExtension>::getSerializedIR(std::string& buildFlags,
-                                                                                IR& irModel,
-                                                                                const Config& config) const {
+std::vector<uint8_t> LevelZeroCompilerInDriver<TableExtension>::getSerializedIR(
```

Comment:
Existing code, but same version retreival is implemented in createGraph above.
I would suggest to extract the version retrieval into separate function and remove getSerializedIR in favor to direct call to serializeIR

Reply:
Removed getSerializedIR and replaced with call of serializeIR.